examples/fluids/navierstokes.c

```diff
@@ -1077,7 +1079,8 @@ int main(int argc, char **argv) {
                        "    DoFs per node                      : %D\n"
                        "    Global nodes                       : %D\n"
                        "    Owned nodes                        : %D\n",
-                       comm_size, box_faces_str, usedresource,
+                       comm_size, problemTypes[problemChoice],
```

Comment:
Should be under the heading
                       "  Problem:\n"
maybe?

Reply:
I don't think it needs a dedicate heading since there is not other option in this case. 
To me having 
```
                       "  Problem:\n"
                       "    Problem Name                       : %s\n"
```
only with the name, it would look a bit awkward. But if you really like it that way, I can change it of course

Reply:
It would be problem name + stabilization type under the heading though?

Reply:
Ah ok, now I see what you meant

Reply:
Looking at my original comment, I probably should have used more words. It's time for some tea.